app/jobs/autotest_cancel_job.rb

```diff
@@ -1,6 +1,12 @@
 class AutotestCancelJob < ApplicationJob
   queue_as MarkusConfigurator.autotest_cancel_queue
 
+  def self.on_complete_js(_status)
+    'window.BatchTestRunTable.fetchData'
```

Comment:
I don't see how this could work right now? I think you need to bind the component to a variable in https://github.com/MarkUsProject/Markus/blob/master/app/views/assignments/batch_runs.html.erb#L4.

Reply:
I don't think you actually added this.